src/server/api/index.js

```diff
@@ -25,6 +28,54 @@ router.get('/platformcontext', (req, res) => {
   }
 });
 
+// Gets library view stats
+router.get('/analytics', (req, res) => {
+  try {
+    stats.getStats().then(result => {
+      res.send(result);
+    });
+  } catch (err) {
+    console.log(err);
+    return res.status(400).send(err);
+  }
+});
+
+// Adds research or reserve view
+router.get('/addview/:type/:srs/:student', (req, res) => {

```

Comment:
@finnzink You shouldn't have to pass student in the parameter. You should be able to get it from the idToken: https://cvmcosta.me/ltijs/#/provider?id=idtoken

You would want to do that because else someone can abuse the API and pass in arbitrary values